src/__mocks__/store.js

```diff
@@ -2,8 +2,8 @@ import thunk from 'redux-thunk';
 import configureMockStore from 'redux-mock-store';
 import initialState from '../store/initialState';
 
-export const middleware = [thunk];
-export const mockStore = configureMockStore(middleware);
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
```

Comment:
@luc-tuyishime, can you please export this, as at some points we may need to mock a different store in a given test file

Reply:
Changes applied